plugins/inputs/sqlserver/azuresqlqueries.go

```diff
@@ -1248,7 +1248,8 @@ BEGIN
 	FETCH NEXT FROM dbCursor INTO @dbName;
 
 	WHILE @@Fetch_Status=0 BEGIN
-		SET @CurrSqlText = N''USE ''+ QUOTENAME(@dbName ,''"'') + @queryText
+		SET @CurrSqlText = N''USE ''+ QUOTENAME(@dbName ,''['') 
+				+ @queryText
```

Comment:
You don't need the second argument for QUOTENAME at all, default is []. 

I meant `QUOTENAME(@dbName) + CONCAT(CHAR(13), CHAR(10)) + @queryText`, sorry if I wasn't clear.

Reply:
Done.